Samples/HelloWorld.NetCore/src/HelloWorld.Grains/HelloWorld.Grains.csproj

```diff
@@ -4,7 +4,7 @@
   </PropertyGroup>
   <ItemGroup>
     <PackageReference Include="Microsoft.Orleans.Core.Abstractions" Version="$(OrleansPackageVersion)" />
-    <PackageReference Include="Microsoft.Orleans.OrleansCodeGenerator.Build" Version="$(OrleansPackageVersion)" />
+    <PackageReference Include="Microsoft.Orleans.OrleansCodeGenerator.Build" Version="$(OrleansPackageVersion)" PrivateAssets="All" />
```

Comment:
These are not strictly required unless you plan to pack these projects, or do something special with the deployment, but they are logically correct anyway.
In the future this might get inferred by NuGet directly. See https://github.com/NuGet/Home/issues/4125